### DISCUSSION TITLE
How are expo compatible versions determined?

### DISCUSSION THREAD

**Discussion by rtturner** [software-mansion/react-native-screens#1958] [Q&A] [answered]
👋 I'm looking to find out how the compatible version with expo is determined. `react-native-screens@3.27.0` has some fixes I'd like to include but `expo-doctor` warns that this version isn't compatible with the latest expo sdk (`49.0.16`). Latest compatible version is `3.22.0`.

Trying to find out how this communication between expo and third party libraries works.

Cheers + Thanks

**Comment by tboba** [comment, marked as answer]
Hi @rtturner! 👋
Right now the newest Expo version supports react-native-screens 3.22.0. Unfortunately, in Expo Go you cannot use any other version, as Expo Go comes with a pre-compiled native code. However, you can use newer/older version of react-native-screens by creating custom development build - you can find more about it [here](https://docs.expo.dev/workflow/customizing/#add-custom-native-code-with-development-builds).

In case of any questions, please let me know.
Cheers!
